content/en/docs/ops/diagnostic-tools/multicluster/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Troubleshooting Multi-Cluster
+description: Describes tools and techniques to diagnose issues with multi-cluster and multi-network installations.
+weight: 20
+keywords: [debug,multi-cluster,multi-network,envoy]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This page describes how to troubleshoot issues with Istio deployed to Virtual Machines.
+Before reading this, you should take the steps in [Multicluster Installation](/docs/setup/install/multicluster/)
+and read the [Deployment Models](/docs/ops/deployment/deployment-models/) guide.
+
+## Cross-Cluster LoadBalancing
+
+The most common, but also broad problem with multi-network installations is that cross-cluster load balancing doesn’t work. Usually this manifests itself as only seeing responses from the cluster-local instance of a Service:
+
+{{< text bash >}}
+$ for i in $(seq 10); do kubectl --context=$CTX_CLUSTER1 -n sample exec sleep-dd98b5f48-djwdw -c sleep -- curl -s helloworld:5000/hello; done
+Hello version: v1, instance: helloworld-v1-578dd69f69-j69pf
+Hello version: v1, instance: helloworld-v1-578dd69f69-j69pf
+Hello version: v1, instance: helloworld-v1-578dd69f69-j69pf
+...
+{{< /text >}}
+
+When following the guide to [verify multicluster installation](https://istio.io/latest/docs/setup/install/multicluster/verify/)
+we would expect both `v1` and `v2` responses, indicating traffic is going to both clusters.
+
+There are many possible causes to the problem:
+
+### Locality Load Balancing
+
+Ensure that locality [load balancing](/docs/tasks/traffic-management/locality-load-balancing/failover/#configure-locality-failover) is not configured or the clusters are in the same locality (region/zone). Locality LB always prefers nearby services, so keeping requests in the same-cluster is working as intended in that case.
+
+### Trust Configuration
+
+**Plug-in Certs:**
+
+You can follow the [Plugin CA Certs](/docs/tasks/security/cert-management/plugin-ca-cert/) guide, ensuring to run
+the steps for every cluster.
+
+To verify certs are configured correctly, you can compare the root-cert in each cluster:
+
+{{< text bash >}}
+$ diff \
+   $(kubectl --context="${CTX_CLUSTER1}" -n istio-system get secret cacerts -ojsonpath='{.data.root-cert\.pem}') \
+   $(kubectl --context="${CTX_CLUSTER1}" -n istio-system get secret cacerts -ojsonpath='{.data.root-cert\.pem}')
+{{< /text >}}
+
+
+### Step-by-step Diagnosis
+
+The following steps assume you're following the [HelloWorld verification](https://istio.io/latest/docs/setup/install multicluster/verify/).
```

Comment:
I think this should be a relative link. Otherwise, different releases could get out of sync